compiler/src/dotty/tools/dotc/core/TypeErasure.scala

```diff
@@ -147,6 +147,12 @@ object TypeErasure {
   def valueErasure(tp: Type)(implicit ctx: Context): Type =
     erasureFn(isJava = false, semiEraseVCs = true, isConstructor = false, wildcardOK = false)(tp)(erasureCtx)
 
+  /** Like value class erasure, but value classes erase to their underlying type erasure */
+  def fullErasure(tp: Type)(implicit ctx: Context): Type =
+    valueErasure(tp) match
```

Comment:
Was there some decision about mixing significant indentation syntax with braces syntax? (this `match` has no braces)

Reply:
I currently use indentation syntax for all code I touch to get a better feeling for it. If we decide not to do indentation, we can rewrite to braces automatically. 

The reason not to convert to indentation wholesale is that is would destroy commit history.